voluptuous/validators.py

```diff
@@ -273,11 +273,11 @@ def _exec(self, funcs, v, path=None):
 Or = Any
 
 class Union(_WithSubValidators):
-    """Use the first validated value among those selected by discrminant.
+    """Use the first validated value among those selected by discriminant.
 
     :param msg: Message to deliver to user if validation fails.
-    :param discriminant(value, validators): Returns the filtered list of validators based on the value
-    :param kwargs: All other keyword arguments are passed to the sub-Schema constructors.
+    :param discriminant(value, validators): Returns the filtered list of validators based on the value.
+    :param kwargs: All other keyword arguments are passed to the sub-schema constructors.
```

Comment:
This was capitalised because it was referencing the `Schema` type, but I don't think it matters 🤷‍♂️

Reply:
I assumed that this was the case, but then still opted for lower-case since in basically all comments it is in lower case (although of course one could argue that there it is not referencing the type but rather the regular English word).